build-system/app.js

```diff
@@ -1299,7 +1299,7 @@ app.use('/shadow/', (req, res) => {
     : `${path.dirname(url)}/`;
 
   const viewerHtml = renderShadowViewer({
-    src: req.url.replace(/^\//, ''),
+    src: '//' + req.hostname + '/' + req.url.replace(/^\//, ''),
```

Comment:
@calebcordry Can you review this change since you originally added this code?

Reply:
LGTM

Reply:
Cherry-picked to another PR: #24066.